generator/src/main/java/com/scottlogic/deg/generator/generation/fieldvaluesources/MustContainsFieldValueSource.java

```diff
@@ -0,0 +1,84 @@
+package com.scottlogic.deg.generator.generation.fieldvaluesources;
+
+import com.scottlogic.deg.generator.utils.ConcatenatingIterable;
+import com.scottlogic.deg.generator.utils.RandomNumberGenerator;
+
+import java.util.*;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class MustContainsFieldValueSource implements FieldValueSource {
```

Comment:
Add java doc

This type may not need to implement this interface, it's only used by the DeDuplicating source